system-parachains/bridge-hubs/bridge-hub-polkadot/src/bridge_to_ethereum_unstuck.rs

```diff
@@ -85,6 +90,8 @@ impl OnRuntimeUpgrade for UnstuckSnowbridge {
 	#[cfg(feature = "try-runtime")]
 	fn post_upgrade(_: Vec<u8>) -> Result<(), TryRuntimeError> {
 		frame_support::ensure!(!is_bridge_stuck(), "Snowbridge is still stuck.");
+		frame_support::ensure!(is_bridge_upgraded(), "Snowbridge is not migrated.");
```

Comment:
I think this is to create false negatives (errors) later down the line when the `try-runtime` tests fork BridgeHub after the relayer has imported new headers, and the tests in `is_bridge_upgraded` fail